### ISSUE TITLE
DOI fetcher: when entry is received, a cleanup should be done

### ISSUE THREAD

**Issue by koppor** [koppor/jabref#8]
When fetching a DOI (e.g., `http://dx.doi.org/10.1145/2684200.2684282`), the result contains

    DOI                      = {10.1145/2684200.2684282},
    URL                      = {http://dx.doi.org/10.1145/2684200.2684282}

The URL should be removed. Maybe, just the DOI cleanup functionality should be called?

**Comment by oscargus** [comment]
I think we should create a way to automatically run selected clean ups on import. At the moment only some of the importers run some of the clean ups (more or less) automatically. Which leads to the serialization discussion.

Personally, I'm not fully convinced that the above is automatically bad. I see some reasons that it is bad, but assuming that the bibtex file may be used by less intelligent tools than JabRef it can make sense to have both in a save bibtex file.

**Comment by koppor** [comment]
Yeah, maybe an autocleanup would make sense. Triggering the `auto save actions` also when importing an entry. I am strong for having a clean `.bib` file (BC) and create a separate file for dumb tools (BW). See https://github.com/JabRef/jabref/issues/160 for a discussion.

**Comment by koppor** [comment]
Result as of today:

```bibtex
@inproceedings{Kpke2014,
  doi = {10.1145/2684200.2684282},
  url = {https://doi.org/10.1145/2684200.2684282},
  year = {2014},
  publisher = {{ACM} Press},
  author = {Julius K\"{o}pke and Johann Eder and Markus K\"{u}nstner},
  title = {Top-Down Design of Collaborating Processes},
  booktitle = {Proceedings of the 16th International Conference on Information Integration and Web-based Applications {\&} Services - {iiWAS} {\textquotesingle}14}
}
```
